src/main/java/org/spongepowered/common/mixin/core/network/MixinNetHandlerPlayServer.java

```diff
@@ -95,6 +103,8 @@
 
     private Location lastMoveLocation = null;
 
+    private final Map<String, ResourcePack> sentResourcePacks = new HashMap<String, ResourcePack>();
```

Comment:
@simon816 That would be a change from the field declarations across all mixins, which to be honest, doesn't need to happen. Regardless over personal preference, making mixins become conforming is much better than suggesting yet another formatting and style change, especially when @Mumfrey is the code style/format manager across the implementations and API.